src/state/nftMarket/reducer.ts

```diff
@@ -29,6 +30,16 @@ export const fetchNftsFromCollections = createAsyncThunk<NFT[], string>(
   },
 )
 
+export const fetchUserNfts = createAsyncThunk<NftSubgraphEntity[], { account: string; tokenIds: number[] }>(
+  'nft/fetchUserNfts',
+  async ({ account, tokenIds }) => {
+    // TODO: This should probably replace the Collectibles state and combine this market data (NftSubgraphEntity) with the fetchWalletNfts data
```

Comment:
We currently store detail about the NFTs a user owns in collectibles state. This probably ought to be migrated, I just wanted to get these first steps PRed first for feedback.